pkg/ccl/backupccl/restore_job.go

```diff
@@ -1161,6 +1161,10 @@ func createImportingDescriptors(
 	// Set the new descriptors' states to offline.
 	for _, desc := range mutableTables {
 		desc.SetOffline("restoring")
+
+		// Remove any LDR Jobs from the table descriptor, ensuring schema changes
+		// can be run on the table descriptor.
+		desc.LDRJobIDs = desc.LDRJobIDs[:0]
```

Comment:
nit: I think `= nil` is a clearer signal of intention to blank it; `= [:0]` is typically associated with an expectation of reuse of the allocated capacity.

Reply:
Done.